lib/core/src/chain/liquid.rs

```diff
@@ -67,33 +56,15 @@ pub trait LiquidChainService: Send + Sync {
     ) -> Result<Transaction>;
 }
 
-#[derive(Deserialize)]
-struct EsploraTx {
-    txid: Txid,
-    status: Status,
-}
-
-#[derive(Deserialize)]
-struct Status {
-    block_height: Option<i32>,
-    block_hash: Option<BlockHash>,
-}
-
 pub(crate) struct HybridLiquidChainService {
```

Comment:
I just noticed this: was the `Hybrid` part of this name related to the fact that it could use either esplora or electrum? If so, we can rename it now :)

Reply:
It was originally named like this as it extends the electrum client, there is a `HybridBitcoinChainService` also. But yeah "Hybrid" is not the best name

Reply:
Ok, then it's not something for this PR